shared/js/background/email-utils.es6.js

```diff
@@ -6,7 +6,7 @@ let attempts = 1
 
 const fetchAlias = () => {
     // if another fetch was previously scheduled, clear that and execute now
-    browser.alarms.get(REFETCH_ALIAS_ALARM, () => browser.alarms.clear(REFETCH_ALIAS_ALARM))
+    browser.alarms.clear(REFETCH_ALIAS_ALARM)
```

Comment:
Had to change this, otherwise the new code threw. Looks like the polyfill doesn't support callbacks. Anyway, it turns out you don't need to call `.get` before you call `.clear`. The call just returns `false` if the alarm was not previously set.

Reply:
Ah yeah sorry this was done elsewhere, it should have used `await` instead of the callback.